app/controllers/ballots_controller.rb

```diff
@@ -47,7 +52,7 @@ def load_ballot
 
   def load_all_beers
     @beers =
-      if brewery = Poll.current.brewery.presence
+      if brewery = Poll.current.try(:brewery)
```

Comment:
Wouldn't Poll.current.brewery return nil? :trollface: Don't need the presence check.